examples/02-plot/point-picking.py

```diff
@@ -0,0 +1,21 @@
+"""
+.. _point_picking_example:
+
+Picking points on a mesh
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+"""
+
+# sphinx_gallery_thumbnail_number = 2
+import pyvista as pv
+
+###############################################################################
+
+sphere = pv.Sphere()
+
+p = pv.Plotter()
+sphere_actor = p.add_mesh(sphere, pickable=False)
+
+p.enable_point_picking(pickable_window=False)
+sphere_actor.SetPickable(True)
```

Comment:
I'm not a huge fan of this... I'd like to avoid having any examples directly utilize the underlying VTK methods if possible (albeit, one of PyVista's selling points is direct access back to VTK when you want to do something more sophisticated).

Either way, this doesn't feel all that Pythonic/PyVista-ish to me

Reply:
Yeah it's not particularly Pythonic. How about creating a method `p.set_pickable_actors` that takes a list of vtk actors to make pickable? Also, should the fine grained control of pickable actors be addressed by a separate issue/PR?

Reply:
Hi @banesullivan , I've added a new property `pickable_actors` to the `BasePlotter`. It can be used to set and retrieve the pickable actors of a given plotter. This means the 3d window can be made pickable independently of which actors are pickable, e.g. if a `limit_to_actors` argument were added to `enable_point_picking`, then it would not be possible to make the 3d window pickable and all actors unpickable. I've also added tests for the new property. Do you think this is a reasonable approach to take? Also, I wasn't sure where to add things, so let me know if you want me to move stuff around.